doc/service_onboarding_guide.md

```diff
@@ -0,0 +1,24 @@
+Service Onboarding Guide
+======
+
+This article aims to provide a guide for service team to request a feature in Azure CLI.
```

Comment:
There's an onboarding guideline already. pls clearly call out audience to avoid confusion. This doc is for service teams whose CLI commands owned by Central CLI team.

Reply:
OK